### PR TITLE
feat(handlers): add startTyping()

### PR DESCRIPTION
**Pull request by itohatweb**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [x] Code changes have been tested against the Discord API, or there are no code changes

**Semantic versioning classification:**

- [x] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/api/handlers/channel.ts [hunk 1]
```diff
@@ -122,6 +122,15 @@ export async function getPins(channelID: string) {
   return Promise.all(result.map((res) => structures.createMessage(res)));
 }
 
+/** 
+ * Trigger a typing indicator for the specified channel. Generally bots should **NOT** implement this route. 
+ * However, if a bot is responding to a command and expects the computation to take a few seconds, 
+ * this endpoint may be called to let the user know that the bot is processing their message
```
Comment (ayntee) [inline src/api/handlers/channel.ts:128]:
```suggestion
 * this endpoint may be called to let the user know that the bot is processing their message.
```
Nitpick: add period for consistency

**Review by ayntee — APPROVED** [review]

File: src/api/handlers/channel.ts [hunk 2]
```diff
@@ -122,6 +122,15 @@ export async function getPins(channelID: string) {
   return Promise.all(result.map((res) => structures.createMessage(res)));
 }
 
+/** 
+ * Trigger a typing indicator for the specified channel. Generally bots should **NOT** implement this route. 
+ * However, if a bot is responding to a command and expects the computation to take a few seconds, 
+ * this endpoint may be called to let the user know that the bot is processing their message.
+ */
+export function triggerTypingIndicator(channelID: string) {
```
Comment (Skillz4Killz) [inline src/api/handlers/channel.ts:130]:
```suggestion
export function startTyping(channelID: string) {
```

Comment (Skillz4Killz) [reply src/api/handlers/channel.ts:130]: my only gripe with this is the name

Comment (itohatweb) [reply src/api/handlers/channel.ts:130]: Discord automatically stops typing indicator after some time. So imo start is not a good name.

Comment (ayntee) [reply src/api/handlers/channel.ts:130]: `Start` does not imply "forever" or "never-ending", it just means begin something with indefinite time to end,

**Review by ayntee — APPROVED** [review]
LGTM! :heart:

**Pull request merged**